### PR TITLE
[all hosts] Scrub versions

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Includes some formatting, terminology, and capitalization cleanup.

### CODE REVIEW

File: docs/testing/debug-desktop-using-edge-chromium.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ This debugging mode is dynamic, allowing you to set breakpoints while code is ru
 - [Visual Studio Code](https://code.visualstudio.com/)
 - [Node.js (version 10+)](https://nodejs.org/)
 - Windows 10, 11
-- A combination of platform and Office application that supports Microsoft Edge with WebView2 (Chromium-based) as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your version of Microsoft 365 is earlier than 2101, you will need to install WebView2. Use the instructions for installing it at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/).
+- A combination of platform and Office application that supports Microsoft Edge with WebView2 (Chromium-based) as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your version of Microsoft 365 subscription Office is earlier than 2101, you will need to install WebView2. Use the instructions for installing it at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/).
```
Comment (alison-mk) [inline docs/testing/debug-desktop-using-edge-chromium.md:26]: The new phrase seems a little awkward, should it be this? 
```suggestion
- A combination of platform and Office application that supports Microsoft Edge with WebView2 (Chromium-based) as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your version of the Microsoft 365 subscription for Office is earlier than 2101, you will need to install WebView2. Use the instructions for installing it at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/).
```

File: docs/develop/develop-overview.md [hunk 2]
```diff
@@ -78,7 +78,7 @@ An Office Add-in can use the Office JavaScript APIs to interact with content in
 
 #### Explore APIs with Script Lab
 
-Script Lab is an add-in that enables you to explore the Office JavaScript API and run code snippets while you're working in an Office program such as Excel or Word. It's available for free via [AppSource](https://appsource.microsoft.com/product/office/WA104380862) and is a useful tool to include in your development toolkit as you prototype and verify the functionality you want in your add-in. In Script Lab, you can access a library of built-in samples to quickly try out APIs or even use a sample as the starting point for your own code.
```
Comment (alison-mk) [inline docs/develop/develop-overview.md:81]: Why remove the URL?

**Review by alison-mk — APPROVED** [review]
Approved with one question and one suggestion (both minor).

File: docs/develop/develop-overview.md (hunk 2 above)
Comment (ElizabethSamuel-MSFT) [reply docs/develop/develop-overview.md:81]: We have 2 AppSource links for Script Lab now (one for WXP and the other for Outlook). It seemed like the best bet to have the reader go to the article, see the options, and determine which they want to install. I'm open to suggestions to include both links here directly, or just let me know if I'm overthinking it.

File: docs/testing/debug-desktop-using-edge-chromium.md (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply docs/testing/debug-desktop-using-edge-chromium.md:26]: the thing is the new phrasing reads to me like we're talking about the version of the subscription, which isn't the case. The message I'm trying to send is about the version of Office downloaded/installed from a Microsoft 365 subscription. Thoughts?

File: docs/develop/develop-overview.md (hunk 2 above)
Comment (alison-mk) [reply docs/develop/develop-overview.md:81]: That makes sense

File: docs/testing/debug-desktop-using-edge-chromium.md (hunk 1 above)
Comment (alison-mk) [reply docs/testing/debug-desktop-using-edge-chromium.md:26]: I see your point. The phrase "Microsoft 365 subscription Office" seems like a modifier stack to me ([style guide link](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=26907)) — I think it's hard to tell that the important part is the Office version. Could we put Office and version closer together in the sentence? Perhaps like this: 

```suggestion
- A combination of platform and Office application that supports Microsoft Edge with WebView2 (Chromium-based) as explained in [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md). If your version of Office is earlier than 2101 (with a Microsoft 365 subscription), you will need to install WebView2. Use the instructions for installing it at [Microsoft Edge WebView2 / Embed web content ... with Microsoft Edge WebView2](https://developer.microsoft.com/microsoft-edge/webview2/).
```

**Pull request merged**